CRM/Upgrade/Incremental/sql/4.7.22.mysql.tpl

```diff
@@ -2,5 +2,14 @@
 
 {include file='../CRM/Upgrade/4.7.22.msg_template/civicrm_msg_template.tpl'}
 
+-- CRM-20816: Add CiviCase settings
+
+SELECT @civicaseAdminId := id FROM civicrm_navigation WHERE name = 'CiviCase' AND domain_id = {$domainID};
+
+INSERT INTO civicrm_navigation
+(domain_id, url, label, name, permission, permission_operator, parent_id, is_active, has_separator, weight)
+VALUES
+({$domainID}, 'civicrm/admin/setting/case?reset=1', '{ts escape="sql" skip="true"}CiviCase Settings{/ts}', 'CiviCase Settings', NULL, 'AND', @civicaseAdminId, '1', NULL, 1);
```

Comment:
Well, I wondered about that and tested -- as you might expect/hope, the two items with `weight==1` float to the top next to each other. The current code seems to work.

Going the other way, it didn't seem like a good idea to manually set the weight of each of the other items -- because they could have been manually rearranged/supplemented/removed, and changing them seemed edgy.

But on second thought... if it really matters... I guess this particular situation could be addressed by incrementing all the `weight`s. Ex:

```
UPDATE civicrm_navigation 
SET weight = weight+1 
WHERE domain_id = {$domainID} AND parent_id = @civicaseAdminId;

INSERT INTO civicrm_navigation ...;
```